shared/constants/chat2/meta.js

```diff
@@ -80,6 +80,57 @@ export const unverifiedInboxUIItemToConversationMeta = (
   })
 }
 
+export const metaReceivedErrorToConversationMeta = (e: RPCChatTypes.InboxUIItemError, username: ?string) => {
```

Comment:
we have a helper to consume the unverified item, why not just call that

Reply:
The reducer was pulling `rekeyers` out of the error which the existing helper (`unverifiedInboxUIItemToConversationMeta`) doesn't do, so I was hesitant to toss that data. I tried using `unverifiedInboxUIItemToConversationMeta` too and it seemed to work, but I didn't test it extensively